### DISCUSSION TITLE
change VtkRemoteLocalView to VtkLocalView

### DISCUSSION THREAD

**Discussion by yjtwinpark** [Kitware/trame#213] [Q&A] [answered]
My example files are too big, playing animation is so slow.
I want to improve speed that.

Let me know how to change VtkRemoteLocalView to VtkLocalView.
When I just replace VtkRemoteLocalView to VtkLocalView in "Time animation paraview" example,

Error messages are shown like followings

ctrl.view_update_image()
raise FUnctionnotImplementedError(self.name)

ctrl.view_update_geometry()
raise FUnctionnotImplementedError(self.name)

**Comment by jourdain** [comment, marked as answer]
`VtkRemoteLocalView` is the only one to have `update_image()/update_geomtry()/update()` while either LocalView or RemoteView only have `update()`.
